src/common/definitions.js

```diff
@@ -125,6 +125,92 @@ const address = (() => {
   };
 })();
 
+const addressBuilder = (useCountryFullName = false) => {
+  const countries = constants.countries.map(object => (useCountryFullName ? object.label : object.value));
+  const countriesWithAnyState = Object.keys(
+    useCountryFullName ? constants.statesWithFullCountryNames : constants.states,
+  ).filter(x => _.includes(countries, x));
+  const countryStateProperties = _.map(
+    useCountryFullName ? constants.statesWithFullCountryNames : constants.states,
+    (value, key) => ({
+      properties: {
+        country: {
+          type: 'string',
+          enum: [key],
+        },
+        state: {
+          type: 'string',
+          enum: value.map(x => x.value),
```

Comment:
It seems like not including `enumNames` would only show lower case full-state names for Canada & Mexico

Reply:
This was copied from the original `address` definition which is being used on most schemas.

```
const address = (() => {
  ...
  const countryStateProperties = _.map(constants.states, (value, key) => ({
    properties: {
      country: {
        type: 'string',
        enum: [key],
      },
      state: {
        type: 'string',
        enum: value.map(x => x.value),
      },
      postalCode: {
        type: 'string',
        maxLength: 10,
      },
    },
  }));
 ...
})();
```